packages/backend/src/controllers/webhooks/handler.ts

```diff
@@ -127,11 +127,19 @@ export default async (request: IRequest, response: Response) => {
   const triggerItem: ITriggerItem = {
     raw: payload,
     meta: {
-      internalId: randomUUID(),
+      internalId:
```

Comment:
can we return the `internalId` from verifyWebhook function. right now only formsg has this function so quite easy to change

Reply:
this is so that we dont have too many formsg specific code here